.gitignore

```diff
@@ -0,0 +1,2 @@
+Maverick-Gliders.git
+.DS_Store/
```

Comment:
Perhaps include the directory `jsonFiles` to this so that all files within it are ignored.

Reply:
I wanted the directory and one sample file to show where the files created will be saved..

Reply:
But ok, will do it